src/main/java/co/cask/gcp/datastore/sink/util/SinkConstants.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package co.cask.gcp.datastore.sink.util;
+
+/**
+ * Datastore Sink constants.
+ */
+public interface SinkConstants {
+
+  String PROPERTY_SERVICE_FILE_PATH = "serviceFilePath";
+  String PROPERTY_PROJECT = "project";
+  String PROPERTY_NAMESPACE = "namespace";
+  String PROPERTY_KIND = "kind";
+  String PROPERTY_KEY_TYPE = "keyType";
+  String PROPERTY_KEY_ALIAS = "keyAlias";
+  String PROPERTY_INDEX_STRATEGY = "indexStrategy";
+  String PROPERTY_ANCESTOR = "ancestor";
+  String PROPERTY_BATCH_SIZE = "batchSize";
+  String PROPERTY_INDEXED_PROPERTIES = "indexedProperties";
+
+  String CONFIG_PROJECT = "mapred.gcd.output.project";
+  String CONFIG_NAMESPACE = "mapred.gcd.output.namespace";
+  String CONFIG_KIND_NAME = "mapred.gcd.output.kind";
+  String CONFIG_USE_AUTOGENERATED_KEY = "mapred.gcd.output.use.autogenerated.key";
+  String CONFIG_BATCH_SIZE = "mapred.gcd.output.batch.size";
+  String CONFIG_SERVICE_ACCOUNT_FILE_PATH = "mapred.gcd.output.service.account.file.path";
+
+  int MAX_BATCH_SIZE = 500;
```

Comment:
There is no available constant in the Datastore Client API for this limit but there is a description in the documentation. Currently batch fails number of entities are more than 500. We need to ensure that we submit not more that max allowed size to avoid partially written data (in case if one batch is over 500 and another is less than 500).